### PR TITLE
Added new z wave binding xml for aeon zw120-a and appropriate product references

### PR DESCRIPTION
**Pull request by ellisbjohns**
Added appropriate xml for the aeon zwave gen5 zw120-a door sensor and appropriate entries to the products.xml file.  All changes confirmed to work.

### CODE REVIEW

**Comment by cdjackson** [comment]
Can you please provide the XML file that OH creates for this device so I can add this into the online database.

Thanks
Chris

File: bundles/binding/org.openhab.binding.zwave/database/aeon/zw120-a.xml [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+Aeotec ZW120 Door/Window Sensor Gen5, base on Engineering Spec 08/01/2016 
+from https://github.com/OpenZWave/open-zwave/blob/master/config/aeotec/zw120.xml
+-->
+<Product>
+	<Model>ZW120-A</Model>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/aeon/zw120-a.xml]: Please change the name to ZW120

File: bundles/binding/org.openhab.binding.zwave/database/products.xml [hunk 2]
```diff
@@ -1000,6 +1004,16 @@
         </Product>
         <Product>
             <Reference>
+                <Type>102</Type>
+                <Id>78</Id>
+            </Reference>
+            <Model>ZW120-a</Model>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/products.xml]: Please change the name to ZW120

File: bundles/binding/org.openhab.binding.zwave/database/aeon/zw120-a.xml (hunk 1 above)
Comment (ellisbjohns) [inline bundles/binding/org.openhab.binding.zwave/database/aeon/zw120-a.xml]: Sure thing.  Will resubmit PR w/ different name.

Comment (ellisbjohns) [inline bundles/binding/org.openhab.binding.zwave/database/aeon/zw120-a.xml]: cdjackson: I've added it to your database.  Hopefully, this is okay.  Thanks again for all your hard work.

**Comment by cdjackson** [comment]
Perfect - thanks very much.

**Pull request merged**